src/ios/CDVUIInAppBrowser.h

```diff
@@ -75,6 +75,7 @@
 @property (nonatomic, strong) IBOutlet UIBarButtonItem* forwardButton;
 @property (nonatomic, strong) IBOutlet UIActivityIndicatorView* spinner;
 @property (nonatomic, strong) IBOutlet UIToolbar* toolbar;
+@property (nonatomic, strong) IBOutlet UIView* backgroundView;
```

Comment:
This isn't really an IBOutlet given that there's no xib/storyboard file nor in reality this is an **I**nterface **B**uilder Outlet. I'll leave it to you as the others are marked as IBOutlets although they aren't